pkg/ipfs/converter_test.go

```diff
@@ -0,0 +1,37 @@
+// VulcanizeDB
+// Copyright © 2019 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package ipfs_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/vulcanize/vulcanizedb/pkg/ipfs/mocks"
+)
+
+var _ = Describe("Converter", func() {
+	Describe("Convert", func() {
+		It("Converts StatediffPayloads into IPLDPayloads", func() {
+			mockConverter := mocks.PayloadConverter{}
+			mockConverter.ReturnIPLDPayload = &mocks.MockIPLDPayload
+			ipldPayload, err := mockConverter.Convert(mocks.MockStatediffPayload)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(ipldPayload).To(Equal(&mocks.MockIPLDPayload))
+			Expect(mockConverter.PassedStatediffPayload).To(Equal(mocks.MockStatediffPayload))
```

Comment:
Yeah absolutely, all the tests need to be finished/reworked!

Reply:
This review was the first time anybody else had eyes on this code, for big changes like this I personally tend to prefer putting off tests until I know the material is generally acceptable to prevent having to double back and redo too many things (but also because I hate writing tests 🙃). This admittedly probably isn't the best approach, since tests can be extremely beneficial to the review process itself.